test-framework/src/main/java/org/elasticsearch/test/ESTestCase.java

```diff
@@ -161,6 +161,25 @@ public static void restoreContentType() {
         Requests.INDEX_CONTENT_TYPE = XContentType.JSON;
     }
 
+    private static boolean TESTS_SEED_PROPERTY_SET = false;
+    @BeforeClass
+    @SuppressForbidden(reason = "sets tests.seed")
```

Comment:
> don't set tests.seed, it is already set.

@rmuir At least not if you run a test in IntelliJ using JUnit integration it is not (the Gradle integration does not work fully at the moment). That's why this code checks to see if the property is set in the first place or not before setting it.